grpc-build/src/builder.rs

```diff
@@ -47,6 +49,11 @@ impl Builder {
         self
     }
 
+    pub fn default_module_name(mut self, name: impl AsRef<str>) -> Self {
```

Comment:
Same doc comment as prost_build
```suggestion
    /// Configures what filename protobufs with no package definition are written to.
    pub fn default_module_name(mut self, name: impl AsRef<str>) -> Self {
```